_posts/2016-10-10-visualise-interruptions-with-a-lego-wall.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+name: visualise-interruptions-with-a-lego-wall 
+title: Visualise interruptions with a LEGO Wall
+date: 2016-10-10 07:00:00 +00:00
+author: Giulia Mantuano
+image:
+   src: /assets/img/custom/blog/2016-10-10-visualise-interruptions-with-a-LEGO-wall/2016-10-10-visualise-interruptions-with-a-LEGO-wall.jpg
+category: agile
+tags:
+- scrum
+- scrum master
+- product owner
+- team
+- retrospective
+- LEGO
+---
+
+
+As part of the [apprentice-craftsmen Programme](https://codurance.com/careers/) at Codurance, I had the privilege to attend the [Certified Scrum Product Owner course](https://skillsmatter.com/courses/152-martine-devos-scrum-product-owner) at Skills Matter. 
+During the two-day course I learnt many techniques to deliver value efficiently, and I’m going to share a peculiar tool to help teams visualise interruptions.
+
+Keeping track of interruptions helps teams improve their productivity during inspect and adapt activities such as daily meetings and retrospectives. In order to turn impediments into opportunities, both development team and stakeholders should collaborate, support and trust each other. These concepts are at the foundation of “Shared responsibility”: everybody is responsible for the success of the business.
+
+Sharing opinions about what is decelerating the team performance is easy in the best-case scenario where we know that no one will assign blame. However, the culture of fear is still widespread in most organisations and people are afraid of being blamed. When people stop sharing their concerns, they lose motivation and start to leave it to the Scrum Master and the Product Owner to fix their problems.
+
+Team members concerns turn into frustrations and become internalised. Impediments become even harder to identify when people are afraid to raise their voice in front of stakeholders.

```

Comment:
`when people are afraid to raise their concerns` OR `when people are afraid to speak up`
"raising their voice" could be read as shouting